qtrouper-core/src/main/java/io/github/qtrouper/Trouper.java

```diff
@@ -190,20 +190,20 @@ public final void publish(Message message, Map<String, Object> headers) throws E
     }
 
     private void publish(Message message, AMQP.BasicProperties properties) throws Exception {
-        log.info("Publishing to {}: {}", queueName, message);
+        log.debug("Publishing to {}: {}", queueName, message);
 
         publishChannel.basicPublish(this.config.getNamespace(), queueName, properties, SerDe.mapper().writeValueAsBytes(message));
 
-        log.info("Published to {}: {}", queueName, message);
+        log.debug("Published to {}: {}", queueName, message);
 
     }
 
     public void sidelinePublish(Message message) throws Exception {
-        log.info("Publishing to {}: {}", getSidelineQueue(), message);
+        log.debug("Publishing to {}: {}", getSidelineQueue(), message);
 
         publishChannel.basicPublish(this.config.getNamespace(), getSidelineQueue(),  new AMQP.BasicProperties.Builder().contentType("text/plain").deliveryMode(2).headers(new HashMap<>()).build(), SerDe.mapper().writeValueAsBytes(message));
 
-        log.info("Published to {}: {}", getSidelineQueue(), message);
+        log.debug("Published to {}: {}", getSidelineQueue(), message);
```

Comment:
Keep the "Published" logs to info, I have kept them to info because I want the logs to say what happened.